api/src/main/java/org/openmrs/module/kenyaemr/reporting/builder/common/ActivePatientSnapshotReportBuilder.java

```diff
@@ -24,9 +25,7 @@
 import org.openmrs.module.kenyaemr.calculation.library.hiv.art.DateOfEnrollmentArtCalculation;
 import org.openmrs.module.kenyaemr.metadata.CommonMetadata;
 import org.openmrs.module.kenyaemr.metadata.HivMetadata;
-import org.openmrs.module.kenyaemr.reporting.calculation.converter.DateArtStartDateConverter;
-import org.openmrs.module.kenyaemr.reporting.calculation.converter.IPTOutcomeDataConverter;
-import org.openmrs.module.kenyaemr.reporting.calculation.converter.SimpleResultDateConverter;
+import org.openmrs.module.kenyaemr.reporting.calculation.converter.*;
```

Comment:
configure your editor to avoid star imports. You will need to push this once it is fixed